appinventor/appengine/src/com/google/appinventor/client/widgets/properties/PropertyEditor.java

```diff
@@ -46,4 +48,12 @@ public void orphan() {
   public void refresh() {
     this.updateValue();
   }
+
+  public void setMultiselectMode(boolean multiselect) {
+    multiselectMode = true;
```

Comment:
@ewpatton I know this has been merged already but I noticed this and I'm curious to know whether or not this logic is intentional? Shouldn't it be `multiselectMode = multiselect`?